src/services/services.ts

```diff
@@ -1794,22 +1794,36 @@ namespace ts {
             return kind === ScriptKind.TS || kind === ScriptKind.TSX;
         }
 
-        function getSemanticClassifications(fileName: string, span: TextSpan): ClassifiedSpan[] {
+        function getSemanticClassifications(fileName: string, span: TextSpan, format?: SemanticClassificationFormat): ClassifiedSpan[] {
             if (!isTsOrTsxFile(fileName)) {
                 // do not run semantic classification on non-ts-or-tsx files
                 return [];
             }
             synchronizeHostData();
-            return ts.getSemanticClassifications(program.getTypeChecker(), cancellationToken, getValidSourceFile(fileName), program.getClassifiableNames(), span);
+
+            const responseFormat = format || SemanticClassificationFormat.Original;
+            if (responseFormat === SemanticClassificationFormat.Original) {
+                return ts.getSemanticClassifications(program.getTypeChecker(), cancellationToken, getValidSourceFile(fileName), program.getClassifiableNames(), span);

```

Comment:
also nit-pick: it would be better to have the original thing be the default Just In Case.

Reply:
Agreed.. Default should be original

Reply:
Right now you can skip including the param, in which case it defaults to the original 👍

Reply:
In a world...

where SemanticClassifications gains another new item, this code starts serving 2020 by default if we forget to update it.

Reply:
Cool, I can do that👍